src/parser.js

```diff
@@ -174,14 +174,15 @@ export default class Parser {
         this.handleSwaggerInfo();
         this.handleSwaggerHost();
         this.handleSwaggerAuth();
+        this.handleSwaggerVendorExtensions(this.api, this.swagger, 'root');
 
         this.handleExternalDocs(this.api, swagger.externalDocs);
 
         this.validateProduces(this.swagger.produces);
         this.validateConsumes(this.swagger.consumes);
 
         const complete = () => {
-          this.handleSwaggerVendorExtensions(this.api, swagger.paths);
+          this.handleSwaggerVendorExtensions(this.api, swagger.paths, 'paths');
```

Comment:
This would be a breaking change for tooling looking for these relations so we need to be careful about rolling out this change. @pksunkara perhaps we should serialise them differently in minim 1.0/0.6 serialisers?

We have a task in our backlog about moving all of the links (such as those in annotations) to be in apielements.org (since many of them are actually 404s). What we could do it update them all and create a profile registry over in API elements. Then in the minim 0.6 serialiser we can replace them to be the existing links.

Another thing to think about is I don't think the current profile is correct given the API Elements 1.0 serialisation rules. Since we're placing an un-refracted object directly in the content of the extension which breaks the rules. Given the following extension:

```yaml
x-extension:
  element: string
```

Tooling would think that this is a string element instead a user-defined object which is some of the problems we was trying to solve in API Elements 1.0. @pksunkara Would love your thoughts, we can probably use this time to do the profile right before wide-adoption of API Elements 1.0.

Reply:
An alternative idea, we could also include both the old style and new style extension element. This allows existing tooling to continue to work (we would have to think of a deprecation plan) and new tooling to directly utilise the newer format. Personally I think my other comment would be better idea long-term but it involves more work short-term (changing minim serialiser and releasing another toolchain version).

Reply:
I like your first comment on this.

- Move the profiles to apielements.org
- Write an RFC about the content of the profiles and implement it
- Convert it into 0.6 serialization.